File: microstartercli/src/main/resources/micronaut/security/jwt/java/JDBC/SecurityClient.java
package ${securityPackage};


import io.micronaut.http.annotation.Body;
import io.micronaut.http.annotation.Post;
import io.micronaut.http.client.annotation.Client;
import io.micronaut.security.authentication.UsernamePasswordCredentials;
import io.micronaut.security.token.render.BearerAccessRefreshToken;

@Client("/api/security/")
public interface SecurityClient {


    @Post("/login")
    public BearerAccessRefreshToken login(@Body  UsernamePasswordCredentials credentials);
}

File: microstartercli/src/main/resources/micronaut/security/jwt/kotlin/JDBC/SecurityClient.kt
package ${securityPackage}

import io.micronaut.http.annotation.Body
import io.micronaut.http.annotation.Get
import io.micronaut.http.client.annotation.Client
import io.micronaut.security.authentication.UsernamePasswordCredentials
import io.micronaut.security.token.render.BearerAccessRefreshToken

@Client("/api/security/")
interface SecurityClient {
    @Get("/login")
    fun login(@Body credentials: UsernamePasswordCredentials?): BearerAccessRefreshToken?
}